### PR TITLE
[config] Add systemctl stop service check function

### PR DESCRIPTION
**Pull request by kuanyu99**
**- What I did**

Add a check function for "systemctl stop $SERVICE" if it returns with error

**- How I did it**

Use "systemctl is-active $SERVICE" to check if the service is dead

**- How to verify it**

Execute "reload config" inside the sonic console

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -516,6 +526,19 @@ def _get_sonic_generated_services(num_asic):
                 generated_services_list.append(line.rstrip('\n'))
     return generated_services_list, generated_multi_instance_services
 
+def _chk_service_stopped(service_name):
```
Comment (jleveque) [inline config/main.py:529]: Suggest renaming to `_wait_for_service_stop`

File: config/main.py [hunk 2]
```diff
@@ -516,6 +526,19 @@ def _get_sonic_generated_services(num_asic):
...
+    # Check the service for 10 times with 5s interval
+    for i in range(10):
+        (srv_active, err) = run_command("systemctl is-active {}".format(service_name))
+        if "inactive" in srv_active:
+            return True
+        time.sleep(5)
```
Comment (jleveque) [inline config/main.py:535]: This will wait up to 50 seconds. Can you please explain the choice of 50 seconds?

File: config/main.py [hunk 3]
```diff
@@ -147,8 +147,16 @@ def execute_systemctl(list_of_services, action):
                 click.echo("Executing {} of service {}...".format(action, service))
                 run_command("systemctl {} {}".format(action, service))
             except SystemExit as e:
-                log_error("Failed to execute {} of service {} with error {}".format(action, service, e))
-                raise
+                if action == SYSTEMCTL_ACTION_STOP:
+                    log_warning("Get error {} when stopping {}. Try to wait it.".format(e, service))
```
Comment (jleveque) [inline config/main.py:151]: Suggest rewording message as follows:

```
"Received error {} when stopping service '{}'. Waiting to see if it stops ...".format(e, service)
```

File: config/main.py [hunk 4]
```diff
@@ -185,6 +193,8 @@ def run_command(command, display_cmd=False, ignore_error=False):
 
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
+    else:
+        return (out, err)
```
Comment (jleveque) [inline config/main.py:197]: Please update the docstring to note that upon success the function returns a tuple containing the contents of stdout and stderr.

File: config/main.py (hunk 2 above)
Comment (kuanyu99) [reply config/main.py:538]: It takes at most 50 seconds when I try to stop each of the services

**Comment by kuanyu99** [comment]
Retest this please

File: config/main.py (hunk 2 above)
Comment (jleveque) [reply config/main.py:538]: Maybe we can increase it to an even 60 seconds to add a buffer, and decrease the sleep to 1 second?

Comment (kuanyu99) [reply config/main.py:538]: Sure. Good point.

**Comment by jleveque** [comment]
Retest this please

**Comment by kuanyu99** [comment]
Retest this please
